### PR TITLE
website: switch to preact

### PR DESCRIPTION
**Pull request by satya164**
I also tried to do it without preact-compat, but had trouble with flow

### CODE REVIEW

File: website/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (thymikee) [inline website/.eslintrc]: `.eslintrc` is deprecated, use `.eslintrc.json` or `.eslintrc.js`

**Comment by thymikee** [comment]
Have you tried `flow-typed create-stub preact`? Also, maybe we should try React 16 RC? It's ~37kB now

**Comment by satya164** [comment]
> Have you tried flow-typed create-stub preact?

It'll only suppress the errors, and the preact code won't typecheck

> Also, maybe we should try React 16 RC? It's ~37kB now

Still much bigger without providing any advantages

**Comment by thymikee** [comment]
If we really want to get rid of example directory, please do so as a separate PR before merging this.

**Comment by satya164** [comment]
I don't want to do multiple PRs and do unnecessary changes to fix errors in example directory. Let's just rebase and merge this.

**Pull request merged**